README.md

```diff
@@ -290,11 +279,13 @@ Follow this guide to run a [hardware exploit](https://icanhack.nl/blog/secoc-key
 
 ### Step 1. Install `TSK Manager`
 
-At home, sitting next to your router, turn on C3X with your phone charger. Ignore the low voltage warning.
+At home, sitting next to your router, turn on Comma with a phone charger. Ignore the low voltage warning.
```

Comment:
<img width="1412" height="243" alt="Image" src="https://github.com/user-attachments/assets/2ef7bf8b-1593-4675-be40-e1624243443f" />

<img width="1389" height="854" alt="Image" src="https://github.com/user-attachments/assets/86ab283b-7939-400f-bfe9-01ec6cb1fc16" />

comma usually refers to their devices without a specific model as "comma device" and not "Comma". I usually try to stick with their style convention.

Reply:
1. The whole `comma` vs `Comma` is very 2005 and mildly infuriating.
2. I got used to calling e-devices by its name without an article (a la "iPhone" not "an iPhone"). I see comma using articles so I'll update the doc accordingly.